src/markdown-pages/add-ons/rke2.md

```diff
@@ -0,0 +1,42 @@
+---
+path: "/docs/add-ons/rke2"
+date: "2021-02-11"
+linktitle: "RKE2 Add-On (Beta)"
+weight: 29
+title: "RKE2 Add-On (Beta)"
+addOn: "rke2"
+---
+
+[RKE2](https://rke2.io/), also known as RKE Government, is a full-conformant Kubernetes distribution from Rancher.
+
+Rather than using the [Kubernetes add-on](/docs/add-ons/kubernetes), which uses kubeadm to install Kubernetes, the RKE2 add-on can be used to install the RKE2 distribution. This distribution includes Kubernetes as well as several add-ons for networking, ingress, and more.
+
+In contrast to K3s, which has deviated from upstream Kubernetes to better support edge deployments, RKE2 maintains closer alignment with upstream Kubernetes.
+
+## Operating System Compatibility
+The RKE2 add-on is currently only supported on CentOS 7 or 8.
```

Comment:
Yes, but technically it does work. We still have CentOS 8 mentioned in our standard kURL docs, just with asterisks next to the versions to indicate they are out of support. If we remove EOL versions, then technically RKE2 is only supported on CentOS 7.9 right now. Perhaps that's better so that we don't start by stating support for an EOL version right off the bat, but technically it'll work on CentOS 8.

Reply:
Hmm... good point. We might technically support it even though it's deprecated. But unless we need to, I'd say we shouldn't bother adding support for CentOS 8. That might be annoying to communicate on https://kurl.sh/docs/install-with-kurl/system-requirements, but I guess we need a different set of system requirements anyway for k3s and RKE2.

Reply:
Okay, I will remove the mention of CentOS 8. If we stick with RKE2, we'll need to support the other OS distros anyway, so we'll move beyond only supporting CentOS 7.9 before long.

> but I guess we need a different set of system requirements anyway for k3s and RKE2.

Yes, this whole thing of adding K3s and RKE2 will be a significant change to the docs since they're so kubeadm focused. Different system requirements needed for each, different default host preflights, etc. We'll fix those things as they come, just commenting that it's a big change to our doc structure since everything has always been kubeadm.